docs/wiki/[User-Guide]-Considerations.md

```diff
@@ -0,0 +1,31 @@
+## Considerations when deploying Enterprise Scale Landing Zones with Terraform
+
+Enterprise Scale Landing Zones makes extensive use of Azure Policy to implement governance.
+Sometimes, certain Azure Policies have compatibility issues or constraints with the Azure Terraform provider.
+This page describes these considerations and provides recommendations where appropriate.
+
+### Deny Subnet Without NSG
+
+The reference architecture assigns the `Deny-Subnet-Without-NSG` policy to the Landing Zones management group.
+In the event that a landing zone subscription is managed by Terraform, this policy have the following unwanted effect:
+
+* Preventing Terraform from deploying `azurerm_subnet` resources due to the way that the Azure Terraform provider interacts with the Azure network resource provider.
+
+#### Recommendation
```

Comment:
If we can change these to just be bold rather than headings, this will address the linting error on this PR